### ISSUE TITLE
jitrolling build is broken

### ISSUE THREAD

**Issue by kunalspathak** [dotnet/runtime#84822]
jit rolling build is broken with error message of "upgrade pip to latest".

### Known Issue Error Message
Fill the error message using [known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-section).

```json
{
  "ErrorMessage": "",
  "BuildRetry": false
}
```

#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|0|0|0|

**Comment by kunalspathak** [comment]
@BruceForstall

**Comment by BruceForstall** [comment]
```
/__w/_temp/48445184-fd1b-4720-8069-2f03978e0806.sh: line 1: pip3: command not found
##[error]Bash exited with code '127'.
```

"Initialize job" shows:
```
Image: build.ubuntu.1804.amd64
Image Version: 2023.0216.171105
```

Maybe someone removed pip3 from the Docker container?

**Comment by BruceForstall** [comment]
This is an "internal" pipeline: https://dev.azure.com/dnceng/internal/_build?definitionId=902&_a=summary

**Comment by BruceForstall** [comment]
This is due to https://github.com/dotnet/runtime/pull/84148

**Comment by EgorBo** [comment]
Since https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/847 is merged, are we waiting for some "update dependencies" PR now?

**Comment by BruceForstall** [comment]
My understanding is we need to wait for an official build of the docker images (https://dev.azure.com/dnceng/internal/_build?definitionId=1004 ?) and then a dotnet/runtime PR to update our YML to use the new images.

**Comment by BruceForstall** [comment]
This is now fixed. I manually triggered a rolling run after the fix in #84863 was mirrored and it worked.
